test/acceptance/behave/components/ngsiv2/entities/update_append_attribute_by_id.feature

```diff
@@ -596,21 +604,37 @@ Feature: update or append an attribute by entity ID in NGSI v2. "POST" - /v2/ent
       | metadatas_type    | alarm    |
       | metadatas_value   | cold     |
     Then verify that receive an "Created" http code
-    And verify that entities are stored in mongo
-    Examples:
-      | entity_id |
-      | room_0    |
-      | room_1    |
-      | room_2    |
+    And verify that an entity is updated in mongo
+    Examples:
+      | entity_id  |
+      | room       |
+      | 34         |
+      | false      |
+      | true       |
+      | 34.4E-34   |
+      | temp.34    |
+      | temp_34    |
+      | temp-34    |
+      | TEMP34     |
+      | house_flat |
+      | house.flat |
+      | house-flat |
+      | house@flat |
+      | habitación |
+      | españa     |
+      | barça      |
+      | random=10  |
```

Comment:
This is literarly "random=10"? Or a construct that generated ten random rows in the examples table?